bemani/utils/frontend.py

```diff
@@ -1,5 +1,4 @@
 import argparse
-
```

Comment:
Repo coding style keeps a newline separating system/3rd party libs and imports from bemaniutils.